production/jaeger-production-template-with-elasticsearch.yml

```diff
@@ -0,0 +1,196 @@
+#
+# Copyright 2017 The Jaeger Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+apiVersion: v1
+kind: List
+items:
+- apiVersion: apps/v1beta1
+  kind: StatefulSet
+  metadata:
+    name: elasticsearch
+  spec:
+    serviceName: elasticsearch
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          app: elasticsearch
+          type: databases
+          availability: restricted
+      spec:
+        containers:
+          - name: elasticsearch
+            image: docker.elastic.co/elasticsearch/elasticsearch:5.6.0
+            imagePullPolicy: Always
+            command:
+              - bin/elasticsearch
+            args:
+              - "-Ehttp.host=0.0.0.0"
+              - "-Etransport.host=127.0.0.1"
+            volumeMounts:
+              - name: data
+                mountPath: /data
+        volumes:
+          - name: data
+            emptyDir: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: elasticsearch
+  spec:
+    clusterIP: None
+    selector:
+      app: elasticsearch
+    ports:
+    - port: 9200
+      name: elasticsearch
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: jaeger-collector
+    labels:
+      app: jaeger
+      jaeger-infra: collector-deployment
+  spec:
+    replicas: 1
+    strategy:
+      type: Recreate
+    template:
+      metadata:
+        labels:
+          app: jaeger
+          jaeger-infra: collector-pod
+      spec:
+        containers:
+        - env:
+          - name: COLLECTOR_ZIPKIN_HTTP_PORT
+            value: "9411"
+          image: jaegertracing/jaeger-collector:0.7
```

Comment:
`0.8` was released already. To keep our sanity, could you please also change the other production template to have the same version as this one?